src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const [fileName, newFileName] = process.argv.slice(2);
+
+function copy() {
+  if (fileName === newFileName) {
+    // eslint-disable-next-line no-console
+    console.log('Name is already exists');
```

Comment:
Consider throwing an error instead of using console.log, e.g., 'throw new Error('Cannot copy a file to itself.');'.